### PR TITLE
mix in -> mixin

### LINKED ISSUE #15727: Misspelling on index?
Index reads: Create "mixin" functionality,  the word "mixin" should be "mix in", 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 646641fc-f698-5633-5f6b-2eb5f0c03d7f
* Version Independent ID: c6e0d402-962e-4b6c-b1f4-4e91eb585692
* Content: [Create mixin types using default interface methods](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/mixins-with-default-interface-methods)
* Content Source: [docs/csharp/tutorials/mixins-with-default-interface-methods.md](https://github.com/dotnet/docs/blob/master/docs/csharp/tutorials/mixins-with-default-interface-methods.md)
* Product: **dotnet-csharp**
* Technology: **csharp-advanced-concepts**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #15727

### CODE REVIEW

**Comment by pkulikov** [comment]
@Youssef1313 "mixin" looks to be correct:
https://en.wikipedia.org/wiki/Mixin
https://docs.scala-lang.org/tour/mixin-class-composition.html

**Comment by Youssef1313** [comment]
> 
> 
> @Youssef1313 "mixin" looks to be correct:
> https://en.wikipedia.org/wiki/Mixin
> https://docs.scala-lang.org/tour/mixin-class-composition.html

Thanks for pointing this. Can you have a look on the current change ?

File: docs/csharp/tutorials/mixins-with-default-interface-methods.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Using default interface members you can extend interfaces with opti
 ms.technology: csharp-advanced-concepts
 ms.date: 10/04/2019
 ---
-# Tutorial: Mix in functionality when creating classes using interfaces with default interface methods
+# Tutorial: Mixin functionality when creating classes using interfaces with default interface methods
```
Comment (pkulikov) [inline docs/csharp/tutorials/mixins-with-default-interface-methods.md]: That sounds like a verb in this case: "mix functionality in"

**Review by BillWagner — APPROVED** [review]
This LGTM @Youssef1313 

I'll :shipit: now.

**Pull request merged**